docs/source/zh/_toctree.yml

```diff
@@ -23,4 +23,8 @@
     title: 使用特定于模型的 API
   - local: custom_models
     title: 共享自定义模型
+  - local: serialization
```

Comment:
It looks like the `serialization.md` file has already been translated: https://huggingface.co/docs/transformers/main/zh/serialization